.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
 
--   repo: .
-    rev: 4b5b03cd59316dc69c1e178fccce0be6101b644c
+-   repo: git://github.com/stratasan/hadolint-pre-commit
```

Comment:
Looks like you're pointing to a remote because you couldn't get the local hooks working. See: https://pre-commit.com/#repository-local-hooks

Reply:
Ah, fantastic. I'll switch back once I've managed to get hadolint to actually run. The problem at the moment is that it needs to run docker from within the docker build agent